Brokerages/Samco/SamcoWebSocketClientWrapper.cs

```diff
@@ -0,0 +1,275 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.IO;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using QuantConnect.Logging;
+using QuantConnect.Util;
+
+namespace QuantConnect.Brokerages.Samco
+{
+    public class SamcoWebSocketClientWrapper : IWebSocket
```

Comment:
We probably don't need this guy? Why not use the `WebSocketClientWrapper`

Reply:
We left this in Zerodha as well. Because of these brokerage specific logs

                            `catch (Exception e)
                            {
                                Log.Error(e, "Error in SamcoWebSocketClientWrapper connection task");
                            }

                            Log.Trace("SamcoWebSocketClientWrapper connection task ended.");`